docs/versionsdescription.rst

```diff
@@ -1,44 +1,79 @@
 .. _versions-description:
 
-AGC Versions
+AGC Analysis Task Versions
 ================================
 
-The below table gives a brief overview of all AGC versions.
+The below table gives a brief overview of the AGC versions. Each version here corresponds to a slightly altered task.
 
-.. list-table:: AGC Versions
-   :widths: 15 16 18 18 15 18
+.. list-table:: Versions
+   :widths: 12 22 22 22 22
    :header-rows: 1
-
+   
    * - Version
      - Datasets
-     - Available Pipelines
      - Cuts
-     - Machine Learning
      - Systematics
-   * - 0.1.0
-     - CMS 2015 Open Data (POET)
-     - Pure ``coffea``; ``coffea`` with ``ServiceX`` processors; ``ServiceX`` followed by ``coffea``
-     - Exactly one lepton with :math:`p_T>25` GeV; at least four jets with :math:`p_T>25` GeV; at least one jet with :math:`b`-tag > 0.5
-     - None
-     - 
-   * - 0.2.0
+     - Machine Learning
+   * - 0
      - CMS 2015 Open Data (POET)
-     - Pure ``coffea``; ``ServiceX`` followed by ``coffea``
      - Exactly one lepton with :math:`p_T>25` GeV; at least four jets with :math:`p_T>25` GeV; at least one jet with :math:`b`-tag > 0.5
+     - :math:`t\bar{t}` sample variations, ``pt_scale`` variations, ``pt_res`` variations, ``btag`` variations, **W + jets** scale variations, luminosity
      - None
-     - 
-   * - 1.0.0
+   * - 1
      - CMS 2015 Open Data (NanoAOD)
-     - Pure ``coffea``; ``ServiceX`` followed by ``coffea``
      - Exactly one lepton with :math:`p_T>25` GeV; at least four jets with :math:`p_T>25` GeV; at least one jet with :math:`b`-tag > 0.5
+     - :math:`t\bar{t}` sample variations, ``pt_scale`` variations, ``pt_res`` variations, ``btag`` variations, **W + jets** scale variations, luminosity
      - None
-     - 
-   * - 2.0.0
+   * - 2 (WIP)
      - CMS 2015 Open Data (NanoAOD)
-     - 
-     - 
+     - Exactly one lepton with :math:`p_T>30` GeV; at least four jets with :math:`p_T>30` GeV; at least one jet with :math:`b`-tag > 0.5 (see :ref:`versions-cuts` for additional cuts)
+     - :math:`t\bar{t}` sample variations, ``pt_scale`` variations, ``pt_res`` variations, ``btag`` variations, **W + jets** scale variations, luminosity
      - BDT to predict jet-parton assignment in :math:`t\bar{t}` events
-     - 
+     
+     
+Description of AGC Tags
```

Comment:
"AGC Tags" is probably too generic, maybe following the title "AGC Analysis Task Versions" above:
```suggestion
Reference Implementation Versions 
```